content/en/platform/corda/4.1/enterprise/api-persistence.md

```diff
@@ -285,219 +285,62 @@ e.g. for Oracle database, prior to version 12.2 the maximum length of table/colu
 
 ## Persisting Hierarchical Data
 
-You may wish to persist hierarchical relationships within states using multiple database tables
+You may wish to persist hierarchical relationships within state data using multiple database tables. 
+In order to facilitate this, you must implement all queries making use of hierarchical relations as native SQL. 
 
-You may wish to persist hierarchical relationships within state data using multiple database tables. In order to facillitate this, multiple `PersistentState`
-subclasses may be implemented. The relationship between these classes is defined using JPA annotations. It is important to note that the `MappedSchema`
-constructor requires a list of *all* of these subclasses.
+Example schemas implementing hierarchical relationships have been implemented below. 
 
-An example Schema implementing hierarchical relationships with JPA annotations has been implemented below. This Schema will cause `parent_data` and `child_data` tables to be
-created.
+For one-to-many scenarios, using the following:
```

Comment:
Isn't this line repeated ? see below (L293, L315)

Reply:
Gosh yes, meant to be one-to-one. Let me refix those.

Reply:
Resolved.